testing/src/main/java/uk/gov/service/payments/commons/testing/pact/consumers/PactProviderRule.java

```diff
@@ -15,17 +15,16 @@
 import java.util.Map;
 import java.util.Optional;
 
-import static au.com.dius.pact.model.PactReader.loadPact;
 import static com.google.common.io.Resources.getResource;
 import static java.lang.String.format;
 import static java.util.Arrays.stream;
 import static uk.gov.service.payments.commons.testing.port.PortFactory.findFreePort;
 
-public class PactProviderRule extends PactProviderRuleMk2 {
+public class PactProviderRule extends au.com.dius.pact.consumer.junit.PactProviderRule {
```

Comment:
Surely an import could be used so this does not need to be fully qualified?

Reply:
Well, not really, unless I rename our class to something that's not the same as pact's class name